src/components/Property/PropertyView.js

```diff
@@ -3,7 +3,7 @@ import { useState, useEffect } from 'react'
 import PropertyDetails from './PropertyDetails'
 import Spinner from '../Spinner/Spinner'
 import ErrorMessage from '../Errors/ErrorMessage/ErrorMessage'
-import { getProperty } from '../../utils/api/repairs/properties/properties'
+import { getProperty } from '../../utils/frontend-api-client/repairs'
```

Comment:
Is there a reason to have this function under `/repairs` rather than `/properties` ?
For example, when we post data from the repairs form or get a work order would we expect this to sit in a different file?

Reply:
Or maybe under `.../repairs/properties` would be a good option?

Reply:
Makes much more sense! [Moved under `/properties`](https://github.com/LBHackney-IT/repairs-hub-frontend/pull/26/commits/35b2262c25a2b8c62917330d05a31eeec5f9fcfa#diff-db659b9529a80962294d5249575a862de8eac3ecee843705b9720b9562182ce5)